### PR TITLE
Raffle page UI

### PR DESCRIPTION
**Pull request by Damola18**
# Improve the UI of the Raffle Card

This PR adds hover effect to the card on the raffle page

image

## Linked issues
closes https://github.com/EveripediaNetwork/issues/issues/2644

### CODE REVIEW

File: src/app/dashboard/page.tsx [hunk 1]
```diff
@@ -23,6 +24,10 @@ import {
   GRAPH_PERIODS,
   StakeGraphPeriod,
 } from '@/data/dashboard-data'
+import {
+  SortAndSumTokensValue,
+  getTreasuryDetails,
+} from '@/utils/treasury-utils'
```
Comment (kesar) [inline src/app/dashboard/page.tsx:30]: this PR has a bunch of unrelated stuff from a different PR. please close and open one with only the raffle fix

Comment (Damola18) [reply src/app/dashboard/page.tsx:30]: > this PR has a bunch of unrelated stuff from a different PR. please close and open one with only the raffle fix

ok sir, will do that now
